### ISSUE TITLE
Add login & onboarding screens

### ISSUE THREAD

**Issue by belgamo** [wendelfreitas/animavita#129]
depends on #128 

#### Login screen
Let's add a [new screen](https://reactnavigation.org/docs/stack-navigator/) so users can type their e-mail and password to log in.

#### Location
The main goal of this app is to match people who want to adopt a pet with people who want to put a pet for adoption. Of course, we don't want to have matches between a person from the North with a person from the South of the country. In order to solve this, let's [capture the user's location](https://docs.expo.dev/versions/latest/sdk/location/) when they first log into our app and store it on the backend so we can use it to filter the adoptions feed.

### Screenshots/examples
| | | | |
:-------------------------:|:-------------------------:|:-------------------------:|:-------------------------:|
![image](https://user-images.githubusercontent.com/19699724/199883729-598cdc73-1ac7-4eb8-9023-ab55c18e1a03.png)| ![image](https://user-images.githubusercontent.com/19699724/199883913-01c62150-5f5d-4e01-ba04-27a8b92aa57d.png) | ![image](https://user-images.githubusercontent.com/19699724/199884070-1e46a95d-ac92-4064-9e85-5410056c556c.png) | ![image](https://user-images.githubusercontent.com/19699724/199884186-1c9d1417-e995-4f7f-9449-ac6b9f95a7d3.png) |

**Comment by joaoGabriel55** [comment]
I can take this

**Comment by Isaac-alencar** [comment]
Hello, I'd like to work on it with @bruno.viveiros as co-author if it's possible

**Comment by belgamo** [comment]
Hey, sure, go ahead! I updated the issue description and removed the secure token storage requirement since it's already done https://github.com/animavita/animavita/pull/165

**Comment by BrunoViveiros** [comment]
Thanks

**Comment by belgamo** [comment]
Hey guys, any update on this? Can I reassign it?

**Comment by Isaac-alencar** [comment]
> Hey guys, any update on this? Can I reassign it?

Hey, sorry for delaying this task. Can I still work on it? I can deliver by the end of the next week. sounds good?

**Comment by belgamo** [comment]
It's done! Thank you guys!
